app/src/components/adminConsole/publishedCups/PublishedCupsHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { SectionHeader } from "../AdminConsoleStyles";
+import Elle from "../../../images/Elle";
+import styled from "styled-components";
+
+export const MiniElleImg = styled(Elle)`
+    height: 66px;
+    width: auto;
+    vertical-align: text-bottom;
+    margin-right: 19px;
+`;
+
+const PublishedCupsHeader: React.FunctionComponent = () => {
+    return (
+        <div id="currentCupHeader">
```

Comment:
Do we actually need this `div`?
The `id` property is incorrect, it's a duplicate of elsewhere.

Reply:
Is now gone